modules/shared_vpc_access/README.md

```diff
@@ -30,7 +30,9 @@ module "shared_vpc_access" {
 | active\_apis | The list of active apis on the service project. If api is not active this module will not try to activate it | `list(string)` | `[]` | no |
 | grant\_services\_security\_admin\_role | Whether or not to grant Kubernetes Engine Service Agent the Security Admin role on the host project so it can manage firewall rules | `bool` | `false` | no |
 | host\_project\_id | The ID of the host project which hosts the shared VPC | `string` | n/a | yes |
+| lookup\_project\_numbers | Whether to look up the project numbers from data sources. If false, `service_project_number` will be used instead. | `bool` | `true` | no |
```

Comment:
I saw that you made this plural - `project numbers from data sources`. Aren't we just using a single data source to look up a single project ?

Reply:
I assume this is defensive naming for a future case where we might have to look up the host project number. Seems reasonable.

Reply:
Right, before it was a bit (lookup_service_project_number). But given @morgante comment above about continuing to support data sources, I imagine we may need to reuse this pattern in other places if this issue comes up, so using a generic/reusable var name will be more consistent for future uses.

Reply:
gotcha, thanks for clarifying.